pkg/services/sqlstore/search_builder.go

```diff
@@ -92,8 +94,17 @@ func (sb *SearchBuilder) ToSql() (string, []interface{}) {
 		LEFT OUTER JOIN dashboard folder on folder.id = dashboard.folder_id
 		LEFT OUTER JOIN dashboard_tag on dashboard.id = dashboard_tag.dashboard_id`)
 
-	sb.sql.WriteString(" ORDER BY dashboard.title ASC" + dialect.Limit(5000))
+	// Default to page 1
+	if sb.page < 1 {
+		sb.page = 1
+	}
+
+	// do not allow limits beyond 5000
+	if sb.limit > 5000 {
+		sb.limit = 5000
```

Comment:
Do you think it would it make sense to have an error also returned here? Seems like this is kinda silent failure and could lead to issues if somebody does not notice it.

Reply:
I was thinking the same when I wrote this, but there is no error return for this function. There is an error for this condition in the api layer so maybe this check can be removed.